proximal/lin_ops/lin_op.py

```diff
@@ -209,7 +209,7 @@ def __str__(self):
     def value(self):
         inputs = []
         for node in self.input_nodes:
-            inputs.append(node.value)
+            inputs.append(np.asfortranarray(node.value, dtype=np.float32))
```

Comment:
I found this minor bug regarding the blur kernel data layout. I will submit the hotfix in a separate PR.